pepper-apis/studybuilder-cli/studies/pancan/patches/trcc-redirect-block-pex.conf

```diff
@@ -0,0 +1,6 @@
+{
+  "is_not_redirect_current": """answers.hasOptionStartsWith("C_BRAIN_")""",
+
+  "is_not_redirect_new": """answers.hasOptionStartsWith("C_BRAIN_", "C_GENITOURINARY_TRCC_RENAL_CELL")""",
```

Comment:
These PEX expression updates are needed so that next sections are NOT visible for tRCC